apps/cmp.c

```diff
@@ -2295,6 +2347,35 @@ static int save_free_certs(STACK_OF(X509) *certs,
     return n;
 }
 
+static int save_crl(X509_CRL *crl,
+                    const char *file, const char *desc)
+{
+    BIO *bio = NULL;
+    int res = 0;
+
+    if (file == NULL)
+        return 1;
```

Comment:
What is 1?

Please Do not use MAGICAL NUMBERS. Please use constants.

IT WOULD BE A NIGHT MARE to determine what is 1 or any other magical numbers.

Reply:
Please do NOT use uppercase and please do not try to introduce new conventions into the codebase that the rest of the code does not use.